erna/qsub.py

```diff
@@ -79,3 +85,99 @@ def get_pending_jobs(job_ids=None):
     if job_ids is not None:
         data = data[data["JOBID"].isin(job_ids)]
     return data[data["state"] == "qw"]
+
+
+def generate_qsub_command(name, queue, jar, xml, inputfile, outputfile, dbpath,
```

Comment:
Is this all maybe redundant since there is a qsub module in automatic processing?